tests/test_notation_sets.py

```
@@ -0,0 +1,182 @@
+    elif item_type == "morphism":
+        return f"M:{params['src']}:{params['tgt']}:{params['label']}"
+    elif item_type == "composition":
+        return f"COMP({params['m1']},{params['m2']})"
+    elif item_type == "identity":
+        return f"ID({params['obj']})"
+    elif item_type == "functor":
+        return f"F:{params['src_cat']}:{params['tgt_cat']}:{params['label']}"
+    elif item_type == "natural_transformation":
+        return f"NAT:{params['src_fun']}:{params['tgt_fun']}:{params['label']}"
+    else:
+        raise ValueError(f"Unknown item type: {item_type}")
```

Pylint (4.0.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)